vyxal/elements.py

```diff
@@ -5858,6 +5859,14 @@ def separate_runl_encode(lhs, ctx: Context):
     return lengths
 
 
+@element("ø%", 1)
+def sha256_hash(lhs, ctx):
+    """Element ø%
+    ...
```

Comment:
Make sure to put an actual element description here

Reply:
Yeah, just realised